### ISSUE TITLE
Cannot deploy to vercel using db-sqlite "Error: Cannot find Module: 'libsql'"

### ISSUE THREAD

**Issue by DanRibbens** [payloadcms/payload#7488]
### Payload Version

3.0.0-beta.73

### Node Version

NA

### Next.js Version

15.0.0-canary.58

### Describe the Bug

https://discord.com/channels/967097582721572934/1268620928426971296/1268966612032028795

### Reproduction Steps

Create a project from: https://github.com/payloadcms/vercel-deploy-payload-postgres after it to use sqlite
Try to deploy to vercel

### Adapters and Plugins

db-sqlite

**Comment by jmikrut** [comment]
Closed via #7490, will be released later today

**Comment by ansbr** [comment]
@jmikrut Hello, I use 3.0.0-beta.77, and I have the same error.
```
Error: Cannot find module 'libsql'
Require stack:
- /var/task/.next/server/app/(payload)/admin/[[...segments]]/page.js
- /var/task/node_modules/.pnpm/next@15.0.0-canary.53_react-dom@19.0.0-rc-fb9a90fa48-20240614_react@19.0.0-rc-fb9a90fa48-2024_es67657svsyc37dl6ildr76u5a/node_modules/next/dist/compiled/next-server/server.runtime.prod.js
- /var/task/___next_launcher.cjs
- /opt/rust/nodejs.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1145:15)
    at /var/task/node_modules/.pnpm/next@15.0.0-canary.53_react-dom@19.0.0-rc-fb9a90fa48-20240614_react@19.0.0-rc-fb9a90fa48-2024_es67657svsyc37dl6ildr76u5a/node_modules/next/dist/compiled/next-server/server.runtime.prod.js:11:26848
    at Module._load (node:internal/modules/cjs/loader:986:27)
    at /opt/rust/nodejs.js:1:11604
    at Function.or (/opt/rust/nodejs.js:1:11978)
    at e.<computed>.te._load (/opt/rust/nodejs.js:1:11574)
    at Module.require (node:internal/modules/cjs/loader:1233:19)
    at u.require (/var/task/node_modules/.pnpm/next@15.0.0-canary.53_react-dom@19.0.0-rc-fb9a90fa48-20240614_react@19.0.0-rc-fb9a90fa48-2024_es67657svsyc37dl6ildr76u5a/node_modules/next/dist/compiled/next-server/server.runtime.prod.js:11:27057)
    at a (/opt/rust/bytecode.js:1:1075)
    at 37418 (/var/task/.next/server/app/(payload)/admin/[[...segments]]/page.js:1:627) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/var/task/.next/server/app/(payload)/admin/[[...segments]]/page.js',
    '/var/task/node_modules/.pnpm/next@15.0.0-canary.53_react-dom@19.0.0-rc-fb9a90fa48-20240614_react@19.0.0-rc-fb9a90fa48-2024_es67657svsyc37dl6ildr76u5a/node_modules/next/dist/compiled/next-server/server.runtime.prod.js',
    '/var/task/___next_launcher.cjs',
    '/opt/rust/nodejs.js'
  ],
  digest: '159834260'
}
```
I tried install to packages `"@libsql/client": "^0.9.0"`, but nothing changes.
This error happens whe I deploying to Vercel.
